pkg/apis/autoscaling/v1alpha1/hvpa_types.go

```diff
@@ -32,18 +32,43 @@ type VpaTemplateSpec struct {
 	// for individual containers. If not specified, the autoscaler computes recommended
 	// resources for all containers in the pod, without additional constraints.
 	// +optional
-	ResourcePolicy *vpa_api.PodResourcePolicy `json:"resourcePolicy,omitempty" protobuf:"bytes,3,opt,name=resourcePolicy"`
+	ResourcePolicy *vpa_api.PodResourcePolicy `json:"resourcePolicy,omitempty" protobuf:"bytes,2,opt,name=resourcePolicy"`
 }
 
+// UpdatePolicy describes the rules on how changes are applied.
+type UpdatePolicy struct {
```

Comment:
I thought we [agreed](https://github.com/gardener/hvpa-controller/pull/19#discussion_r312775612) that we will have two different types for update policy for HPA and VPA. With the current approach, we will have to validate the scenario where `ScaleOut` is given for VPA and `ScaleUp` is given for HPA. Are there new arguments?

Reply:
As discussed offline, we will use `scaleUp` for both HPA and VPA. Will make the changes.